boards/xtensa/m5stack_stamps3/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+.. _m5stack_stamps3:
+
+M5Stack StampS3
+###############
+
+Overview
+********
+
+M5Stack StampS3 is an ESP32-based development board from M5Stack.
+It features the following integrated components:
+
+- ESP32-S3FN8 chip (240MHz dual core)
+- 512KB SRAM
+- 384KB ROM
+- 8MB Flash
+- Wi-Fi
+- Bluetooth
+- User-Button
+
+.. figure:: img/m5stack_stamps3.webp
+        :align: center
+        :alt: M5Stack StampS3
+        :width: 400 px
+
+        M5Stack StampS3 module
+
+Functional Description
+**********************
+
+The following table below describes the key components, interfaces, and controls
+of the M5Stack StampS3 module.
+
++---------------+------------------------------------------------------------------+------------+
+| Key Component | Description                                                      | Status     |
++===============+==================================================================+============+
+|| ESP32-S3FN8  || This MPU-ESP32S3 module provides complete Wi-Fi and Bluetooth   || supported |
+|| module       || functionalities and integrates a 8MB flash.                     ||           |
++---------------+------------------------------------------------------------------+------------+
+|| Status LED   || One user LED connected via :dtcompatible:`worldsemi,ws2812-spi` || supported |
+||              || interface (``led-strip``).                                      ||           |
++---------------+------------------------------------------------------------------+------------+
+|| USB Port     || USB interface. Power supply for the board as well as the        || supported |
+||              || communication interface between a computer and the board.       ||           |
++---------------+------------------------------------------------------------------+------------+
+| User button   | User button (``sw0``)                                            | supported  |
++---------------+------------------------------------------------------------------+------------+
```

Comment:
Hey just noticed this, not a big deal as it renders fine apparently, but what's with the double `|`s? I don't think they are needed? In fact, the only other instance I could find of such an odd instance I could find in-tree is in M5Stack Core2 doc, so my bad for not spotting this before ;-)

Could you please clean it up? FWIW things look good to me otherwise (I wish I would have the hardware at hand to test further though), so here's my virtual +1 :)